controllers/consoleplugin/consoleplugin_objects.go

```diff
@@ -84,6 +85,39 @@ func (b *builder) consolePlugin() *osv1alpha1.ConsolePlugin {
 	}
 }
 
+func (b *builder) consolePluginServiceMonitor() *monitoringv1.ServiceMonitor {
+	return &monitoringv1.ServiceMonitor{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      constants.PluginServiceMonitorName,
+			Namespace: constants.DefaultOperatorNamespace,
```

Comment:
```suggestion
			Namespace: b.namespace,
```

Namespace should be taken from CRD